packages/fast-components-react-base/src/tabs/examples.data.tsx

```diff
@@ -189,29 +192,6 @@ function getTabItemFour(): any {
     };
 }
 
-const detailChildren: any[] = [getTabItemOne(), getTabItemTwo(), getTabItemThree()];
-
-const exampleChildren1: any[] = [
-    {
-        id: tabItemSchema.id,
-        props: {
-            slot: TabsSlot.tabItem,
-            id: "tab01",
-            children: {
-                id: tabSchema.id,
-                props: {
-                    slot: TabsSlot.tab,
-                    ...tabManagedClasses,
-                    children: "tab one - missing panel",
-                },
-            },
-        },
-    },
-    getTabItemTwo(),
-    getTabItemThree(),
-    getTabItemFour(),
-];
-
```

Comment:
I would like to see the original examples as well as the new ones so that the new ones are represented and in the detailData and are the ones at the top of the data array, if that makes sense. It's OK there is duplication, the important thing to see in the PR for validation purposes is that all examples previously are represented, plus a new set of examples for your update.